docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,575 @@
+
+:::
+
+::: tab strapi
+
+```
+strapi install i18n
+```
+
+:::
+
+::::
+
+## Usage with Strapi Content API
+
+The i18n plugin adds new features to the [Content API](/developer-docs/latest/developer-resources/content-api/content-api.md):
+
+- a new [`locale`](#getting-localized-entries-with-the-locale-parameter) parameter to fetch content only for a specified locale
+- the ability to create a localized entry, either [from scratch](#creating-a-new-localized-entry) or [for an existing localized entry](#creating-a-localization-for-an-existing-entry)
+
+### Getting localized entries with the `locale` parameter
+
+The `locale` [API parameter](/developer-docs/latest/developer-resources/content-api/content-api.md#api-parameters) can be used to fetch entries only for a specified locale. It takes a locale code as value (see [full list of available locales](https://github.com/strapi/strapi/blob/features/i18n/packages/strapi-plugin-i18n/constants/iso-locales.json)).
+
+:::tip TIP
+ To fetch content for a locale, make sure it has been already [added to Strapi in the admin panel GUI](/user-docs/latest/settings/managing-global-settings.md#configuring-internationalization-locales).
+:::
+
+The format for a GET request is the following:
+
+```js
+GET /{content-type}?_locale={locale-code}
+```
+
+If the `locale` parameter isn't defined, it will be set to the default locale. `en` is the default locale when i18n plugin is installed, so by default a GET request to `http://localhost:1337/restaurants` will return the same response as a request to `http://localhost:1337/restaurants?_locale=en`.
+
+::: tip TIP
+Another locale can be [set as the default locale](/user-docs/latest/settings/managing-global-settings.md#adding-a-new-locale) in the admin panel.
+:::
+
+<!-- ? Should we be more explicit about what a locale code is? What are the available locales? Is there a list? -->
+<!-- TODO maybe add a link to user guide here if Mégane describes the list of available locales -->
+
+When the i18n plugin is installed, the response to requests includes fields that are specific to internationalization:
+
+- `locale` (string) is the locale code for the content entry
+- `localizations` (array) lists the existing localizations for this content entry; these localizations objects have 3 properties:
+  - `id` (number|string) is the id of the localized content entry
+  - `locale`(string) is the locale code for the localized content entry
+  - `published_at` (string) is the date and time when the localized content entry was [published](/developer-docs/latest/concepts/draft-and-publish.md#publishing-a-draft), in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format
+
+
+:::: tabs
+
+::: tab Request
+
+**Example request**
+
+```js
+GET http://localhost:1337/restaurants?_locale=fr
+```
+
+:::
+
+::: tab Response
+
+**Example response**
```

Comment:
I think there is too many fields display.
To make it easier to understand, I would drastically simplify JSON objects by just keeping 2/3 fields that are interesting. And I would use some restaurant names that come from FoodAdvisor instead of 
```
"name": "Restaurant A",
"slug": "restaurant-a-1",
```

Reply:
Thanks Jim!
I wanted to use real-world examples and I copied/pasted the full response that I had with testing the feature.

But you're right, I'll simplify this and update it with FoodAdvisor-based examples.